src/components/Tooltip/index.tsx

```diff
@@ -61,6 +90,8 @@ export const Tooltip = ({
       <span
         ref={tooltipContentRef}
         className={classNames("common-tooltip-content", {
+          top: position === TOOLTIP_POSITION.top,
+          right: position === TOOLTIP_POSITION.right,
           "position-right": positionRight,
```

Comment:
does it still have effect? let say  set this true and set position to top. what's going to happen?

Reply:
you mean for the `positionRight` right? I think it still have effect it will make the tooltip slightly to the right

for props `position `
when it say to top the tooltip will come at the top ( with arrow point down ), when it set to "right" the tooltip will be on the right with tooltip point to the right